evadb/udfs/chatgpt.py

```diff
@@ -97,21 +97,26 @@ def completion_with_backoff(**kwargs):
         results = []
 
         for prompt, query in zip(prompts, queries):
-            if prompt != "None":
-                query = prompt + ": " + query
+            # if prompt != "None":
+            #     query = prompt + ": " + query
 
             params = {
                 "model": self.model,
                 "temperature": self.temperature,
                 "messages": [
                     {
                         "role": "user",
-                        "content": query,
+                        "content": f"Context to answer the question : {query}",
+                    },
+                    {
+                        "role": "user",
+                        "content": f"Answer the question based on context : {prompt} Answer only from the context, don't generate responses on your own",
```

Comment:
I believe the model generates answers conservatively with temperature = 0. It is good that the query is specific, but perhaps we don't want it to be too lengthy because it takes api tokens. Do we have proof that querying in this format gives better answers? Plus, is there a reason to not let the llm to only answer based on the context? A user may ask very broad questions.

Reply:
While benchmarking, I was observing that sometimes the model was generating answers to the questions on its own and the generated answer doesnt match the info in the given video. Hence i add the restriction to answer based on the context. 

Removed it for now.